manifest/spark-operator-rbac.yaml

```diff
@@ -45,6 +45,9 @@ rules:
 - apiGroups: [""]
   resources: ["events"]
   verbs: ["create", "update", "patch"]
+- apiGroups: ["batch"]
+  resources: ["jobs"]
+  verbs: ["create", "get", "update", "delete"]
```

Comment:
would need list/watch permissions as well for jobs

Reply:
Done.